src/main.py

```diff
@@ -159,6 +159,7 @@ async def main() -> None:
     offset = os.environ["OFFSET"]
     kafka_schema_registry_url = os.environ["KAFKA_SCHEMA_REGISTRY_URL"]
     latiss_knative_serving_url = os.environ["LATISS_KNATIVE_SERVING_URL"]
+    lsstcomcam_knative_serving_url = os.environ["LSSTCOMCAM_KNATIVE_SERVING_URL"]
```

Comment:
I guess we already have a URL defined in the Phalanx config, but this makes me a bit nervous -- trying to support all the instruments in advance has caused us a lot of unnecessary maintenance...